core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/replication-repmgr-Debian.yml

```diff
@@ -0,0 +1,174 @@
+---
+
+- name: Add user to sudoers file
+  lineinfile:
+    path: /etc/sudoers
+    line: 'postgres ALL=(ALL:ALL) NOPASSWD: /bin/systemctl start postgresql, /bin/systemctl stop postgresql, /bin/systemctl restart postgresql, /bin/systemctl reload postgresql'
+    validate: 'visudo -cf %s'
+
+- name: Turn on replica
+  replace:
+    path: /etc/postgresql/10/main/postgresql.conf
+    regexp: "^#wal_level = replica"
+    replace: "wal_level = replica"
+    backup: yes
+
+- name: Set max_wal_senders
+  replace:
+    path: /etc/postgresql/10/main/postgresql.conf
+    regexp: "^[#]?max_wal_senders\\s*=\\s*[0-9]+"
+    replace: "max_wal_senders = {{ specification.additional_components.replication.max_wal_senders | default('5') }}"
+
+- name: Add password for super user in postgresql
+  become_user: postgres
+  postgresql_user:
+    user: "{{ specification.additional_components.replication.user }}"
+    password: "{{ specification.additional_components.replication.password }}"
+    role_attr_flags: LOGIN,SUPERUSER
+
+- name: Add replication user to postgresql
+  become_user: postgres
+  postgresql_user:
+    user: "{{ specification.additional_components.replication.user }}"
+    password: "{{ specification.additional_components.replication.password }}"
+    role_attr_flags: LOGIN,REPLICATION
+
+- name: Add replication user to pg_hba.conf
+  lineinfile:
+    path: /etc/postgresql/10/main/pg_hba.conf
+    line: "host    replication     {{ specification.additional_components.replication.user }}     0.0.0.0/0            md5"
+
+- name: Create pgpass file
+  template:
+    src: pgpass-repmgr.j2
+    dest: /var/lib/postgresql/.pgpass
+    owner: postgres
+    group: postgres
+    mode: 0600
+
+- name: Set Node ID
```

Comment:
It looks like this var is used only for 'Create repmgr config file' task so don't have to be global.

You can set this var in the jinja template using `set` directive or write the task like:
```
- name: Create repmgr config file
  template:
    src: repmgr.conf.j2
    dest: /etc/postgresql/10/main/repmgr.conf
    owner: postgres
    group: postgres
    mode: 0600
  vars:
    node_id: "{{ (groups.postgresql.index(inventory_hostname) + 1) }}"
```